src/utils/fieldUtils/serializeFields.js

```diff
@@ -13,7 +13,8 @@ export default function serializeFields(fields, valueResolver = null) {
 
     if (!isCheckbox && hasEmptyValue) return serialized;
 
-    const defaultValue = isCheckbox ? fieldProps.get('checked') : fieldProps.get('value');
+    const valueProp = fieldProps.get('valueProp');
```

Comment:
`valuePropName`?

left me confused for a sec

Reply:
I like the idea to rename to `valuePropName`.